mroonga.rb

```diff
@@ -25,6 +27,14 @@ class Mroonga < Formula
     depends_on "cmake" => :build
     depends_on "boost" => :build
     depends_on "mysql"
+  elsif build.include?("use-homebrew-mysql56")
+    depends_on "cmake" => :build
+    depends_on "boost" => :build
```

Comment:
I think that MySQL 5.6 does not depend on boost because it does not use `Boost.Geometry`.
Could you try to build without `depends_on "boost" => :build` ?